fix(CortexPlatformCore): render XDR BIOC issue description as readable text in core-get-issues (#44573)
demisto/content: Packs/ApiModules/ReleaseNotes/2_4_26.md

#### Scripts

##### CoreIRApiModule

> - Fixed an issue where the ***core-get-issues*** command returned the description of XDR BIOC issues as a raw structured object instead of readable text, matching the way it is displayed in the Cortex UI.

Reviewer: ```suggestion
- Fixed an issue where the ***core-get-issues*** command returned the description of the *XDR BIOC* issue type as a raw structured object instead of readable text.
```
